### PR TITLE
Expose app_role id

### PR DESCRIPTION
**Pull request by voidcontext**
Expose app_role id so that we can reference it in policies.

### CODE REVIEW

**Comment by mwz** [comment]
This module requires terraform 0.12, so you can drop the interpolation syntax.

**Comment by voidcontext** [comment]
> This module requires terraform 0.12, so you can drop the string interpolation syntax.

@mwz: Removed.

File: output.tf [hunk 1]
```diff
@@ -25,3 +25,7 @@ output "alb_security_group_id" {
 output "cicd_user_name" {
   value = "${aws_iam_user.cicd.name}"
 }
+
+output "ecs_app_role_id" {
+  value = ${aws_iam_role.app_role.id}
```
Comment (mwz) [inline output.tf:30]: Well, you can just use `aws_iam_role.app_role.id`, I don't think that `${aws_iam_role.app_role.id}` is a valid syntax without interpolation.

Comment (voidcontext) [reply output.tf:30]: This is probably correct, as the build that's using this module is failing because of this, I'll open a new PR asap.

Comment (mwz) [reply output.tf:30]: I really dislike the fact that this repo doesn't have any CI checks and doesn't make it easy to catch errors like this.

**Pull request merged**